### PR TITLE
chore: add fullstory

### PR DESCRIPTION
**Pull request by owlas**
Adds the option to integrate with fullstory

### CODE REVIEW

File: packages/backend/src/config/parseConfig.ts [hunk 1]
```diff
@@ -211,6 +217,10 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
             environment:
                 process.env.NODE_ENV === 'development' ? 'development' : mode,
         },
+        fullstory: {
+            orgId: process.env.FULLSTORY_ORG_ID || '',
+            devMode: process.env.NODE_ENV === 'development',
```
Comment (ZeRego) [inline packages/backend/src/config/parseConfig.ts:222]: || lightdashConfig.mode === LightdashMode.DEMO || lightdashConfig.mode === LightdashMode.PR

Comment (ZeRego) [inline packages/backend/src/config/parseConfig.ts:222]:
```suggestion
            devMode: process.env.NODE_ENV === 'development || lightdashConfig.mode === LightdashMode.DEMO || lightdashConfig.mode === LightdashMode.PR',
```

Comment (owlas) [reply packages/backend/src/config/parseConfig.ts:222]: 👍 although as long as we don't set the key in dev/pr then this won't fire anyway!

**Comment by rephus** [comment]
This new flag is breaking e2e tests

We should not load  it if `CI  === true` 

image

**Comment by owlas** [comment]
@rephus fixed!! The error was because I was calling `FS.identify` before it was loaded. Now identify only happens if we load full story. And full story is loaded if there is a valid org_id supplied. So in CI / demo / PRs etc. there will never be full story activated

**Review by rephus — APPROVED** [review]

**Pull request merged**